src/sgame/sg_active.cpp

```diff
@@ -2249,6 +2249,14 @@ void ClientThink( int clientNum )
 
 	ent = g_entities + clientNum;
 	trap_GetUsercmd( clientNum, &ent->client->pers.cmd );
+	if ( ent->client->pers.cmd.flags )
```

Comment:
Make it really a flag instead of using cmd.flags as a boolean?

Reply:
Done